src/models.ts

```diff
@@ -32,3 +70,70 @@ export const baseMetricValues: Record<BaseMetric, BaseMetricValue[]> = {
   [BaseMetric.INTEGRITY]: ['N', 'L', 'H'],
   [BaseMetric.AVAILABILITY]: ['N', 'L', 'H']
 };
+
+export const environmentalMetricValues: MetricValues<
+  EnvironmentalMetric,
+  EnvironmentalMetricValue
+> = {
+  [EnvironmentalMetric.ATTACK_VECTOR]: ['N', 'A', 'L', 'P', 'X'],
+  [EnvironmentalMetric.ATTACK_COMPLEXITY]: ['L', 'H', 'X'],
+  [EnvironmentalMetric.PRIVILEGES_REQUIRED]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.USER_INTERACTION]: ['N', 'R', 'X'],
+  [EnvironmentalMetric.SCOPE]: ['U', 'C', 'X'],
+  [EnvironmentalMetric.CONFIDENTIALITY]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.INTEGRITY]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.AVAILABILITY]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.CONFIDENTIALITY_REQUIREMENT]: ['M', 'L', 'H', 'X'],
+  [EnvironmentalMetric.INTEGRITY_REQUIREMENT]: ['M', 'L', 'H', 'X'],
+  [EnvironmentalMetric.AVAILABILITY_REQUIREMENT]: ['M', 'L', 'H', 'X']
+};
+
+export const temporalMetricValues: MetricValues<
+  TemporalMetric,
+  TemporalMetricValue
+> = {
+  [TemporalMetric.EXPLOITABILITY]: ['X', 'U', 'P', 'F', 'H'],
+  [TemporalMetric.REMEDIATION_LEVEL]: ['X', 'O', 'T', 'W', 'U'],
+  [TemporalMetric.REPORT_CONFIDENCE]: ['X', 'U', 'R', 'C']
```

Comment:
let keep naming and possible values order as they are in spec (https://www.first.org/cvss/v3.0/specification-document#Vector-String and https://www.first.org/cvss/v3.1/specification-document#Vector-String)

```suggestion
  [TemporalMetric.EXPLOIT_CODE_MATURITY]: ['X', 'H', 'F', 'P', 'U'],
  [TemporalMetric.REMEDIATION_LEVEL]: ['X', 'U', 'W', 'T', 'O'],
  [TemporalMetric.REPORT_CONFIDENCE]: ['X', 'C', 'R', 'U']
```